frontends/p4/fromv1.0/converters.cpp

```diff
@@ -546,6 +546,8 @@ class DiscoverStructure : public Inspector {
     explicit DiscoverStructure(ProgramStructure* structure) : structure(structure)
     { CHECK_NULL(structure); setName("DiscoverStructure"); }
 
+    void postorder(const IR::ParserException* ex) override
```

Comment:
Note: this line is currently never executed, since the bison parser never creates a ParserException object.